2019-05-20 15:51:02,794 [DEBUG] Using GPU 0
2019-05-20 15:51:02,815 [DEBUG] CACHEDIR=/home/thinh/.cache/matplotlib
2019-05-20 15:51:02,816 [DEBUG] Using fontManager instance from /home/thinh/.cache/matplotlib/fontlist-v300.json
2019-05-20 15:51:02,869 [DEBUG] Loaded backend agg version unknown.
2019-05-20 15:51:03,776 [DEBUG] update_title_pos
2019-05-20 15:51:03,780 [DEBUG] findfont: Matching :family=sans-serif:style=normal:variant=normal:weight=normal:stretch=normal:size=10.0 to DejaVu Sans ('/home/thinh/miniconda3/envs/ano/lib/python3.6/site-packages/matplotlib/mpl-data/fonts/ttf/DejaVuSans.ttf') with score of 0.050000.
2019-05-20 15:51:14,320 [DEBUG] update_title_pos
2019-05-20 15:51:28,218 [INFO] ==> VIDEO_ROOT_PATH/UCSD_ped1/testing_h5_t0/UCSD_ped1_01.h5
2019-05-20 15:51:28,218 [DEBUG] Predicting using logs/UCSD_ped1/jobs/f5846318-6990-4622-b584-ececf33a54d7/model_snapshot_e298_0.003159.h5
2019-05-20 15:51:36,202 [CRITICAL] Uncaught exception
Traceback (most recent call last):
  File "/home/thinh/github/anomaly-detection-and-localization/start_test.py", line 73, in <module>
    visualize_score=True, visualize_frame=True)
  File "/home/thinh/github/anomaly-detection-and-localization/classifier.py", line 669, in test
    total_err = fusion_matrix(t_err, s_err, type="mul")
  File "/home/thinh/github/anomaly-detection-and-localization/classifier.py", line 584, in fusion_matrix
    return normalize(0.5*matrixA*matrixB)
ValueError: operands could not be broadcast together with shapes (200,160,240) (200,1,160,240,1) 
